docs/version0.10.md

```diff
@@ -62,6 +63,26 @@ For compatibility, the following OpenJDK HotSpot options are now supported by Op
 - [-XX:\[+|-\]HeapDumpOnOutOfMemory](xxheapdumponoutofmemory.md)
 - [-XX:ActiveProcessorCount](xxactiveprocessorcount.md)
 
+### ![Start of content that applies to Java 11 plus](cr/java11plus.png) New JDK 11 features
+
+The following features are supported by OpenJ9:
+
+- [JEP 181](https://openjdk.org/jeps/181): Nest-Based Access Control
+- [JEP 309](https://openjdk.org/jeps/309): Dynamic Class-File Constants
+- [JEP 318](https://openjdk.org/jeps/318): Epsilon: A No-Op Garbage Collector
+- [JEP 320](https://openjdk.org/jeps/320): Remove the Java EE and CORBA Modules
+- [JEP 321](https://openjdk.org/jeps/321): HTTP Client (Standard)
+- [JEP 323](https://openjdk.org/jeps/323): Local-Variable Syntax for Lambda Parameters
+- [JEP 324](https://openjdk.org/jeps/324): Key Agreement with Curve25519 and Curve448
+- [JEP 329](https://openjdk.org/jeps/329): ChaCha20 and Poly1305 Cryptographic Algorithms
+- [JEP 330](https://openjdk.org/jeps/330): Launch Single-File Source-Code Programs
+- [JEP 331](https://openjdk.org/jeps/331): Low-Overhead Heap Profiling
```

Comment:
This should have a note about the restrictions mentions earlier in this topic.

Reply:
Sorry, the restrictions you have mentioned in migrating8to11.md

Reply:
While incorporating this change, we noticed that JEP 331 is discussed in What's new in Version0.10.0 topic as well as What's new Version0.15.1 topic. Is it supposed to be discussed in both? If not, then from which topic should it be deleted?

Thanks!

Reply:
JEP 331 support wasn't available in the 0.10 release, but was added later in 0.15.1